mypy.ini

```diff
@@ -0,0 +1,3 @@
+[mypy]
+ignore_missing_imports = True
+disable_error_code = misc,attr-defined, import
```

Comment:
Let's work on fixing as many warnings/errors as possible in these categories instead of disabling the errors globally. The `misc` category should be easy. It is just a matter of rearranging the variable declarations. A good IDE like PyCharm can probably do this automatically (haven't tested it myself).

Reply:
Fixed all error except the `ignore_missing_imports` which I have no control to fix, since the third party packages that not supporting typing are the cause of this.